migrator/migration.py

```diff
@@ -133,6 +134,14 @@ def disable_migration_service_plan(self):
         for service_plan_visibility_id in self.service_plan_visibility_ids:
             cf.disable_plan_for_org(service_plan_visibility_id, self.client)
 
+    def create_bare_migrator_instance_in_org_space(self):
+        cf.create_bare_migrator_service_instance_in_space(
```

Comment:
Should we check the response? Make sure it's either `in progress` or provisioned (can't recall if that's `provisioned` or `created` or something else)

Reply:
Added!